include/deal.II/grid/tria.h

```diff
@@ -2994,6 +2994,14 @@ class Triangulation : public Subscriptor
    * Return an iterator to a cell of this Triangulation object constructed from
    * an independent CellId object.
    *
+   * @note See the documentation of contains_cell() about which CellId objects
+   * are valid.
+   */
+  cell_iterator
+  create_cell_iterator(const CellId &cell_id) const;
+
+  /**
+   * Check if the triangulation contains a cell with the id @p cell_id.
    * If the given argument corresponds to a valid cell in this triangulation,
    * this operation will always succeed for sequential triangulations where the
```

Comment:
Is 'succeed' still the right word in this new context? I would assume from the name that `contains_cell` will never throw an assertion ('succeed'), but might return either `true` or `false` depending on the specified criterion?